### DISCUSSION TITLE
Blockquotes with Other Elements in combination with the Styling Feature

### DISCUSSION THREAD

**Discussion by RED-CAD-Solutions-AG-Development** [requarks/wiki#6094] [Error / Bug Report] [answered]
I think there is an error in rendering blockquotes when you combine blockquotes with other elements and also want to use the styling feature (is-success, etc.). (See example)

I wanted to include an unordered list in the quote. 

Basically this works, but not in combination with the styling feature. 

If you add an additional line after the unordered list, the styling works. 

<img width="1722" alt="blockquote with other elements and styling feature" src="https://user-images.githubusercontent.com/51949837/214846940-5cb6fe7f-0b9e-4901-bab4-d5d1571c1f7f.png">

Example Code: 

```
> **This doesn't work!**
>
> - Test
> - Test
> - Test
>
{.is-success}

> **This works!**
>
> - Test
> - Test
> - Test
>
>  *With Additional text*.
{.is-success}
```

In this post there might be the same problem → https://github.com/requarks/wiki/discussions/5381

**Comment by NGPixel** [comment, marked as answer]
https://github.com/requarks/wiki/discussions/4732#discussioncomment-4783499

**Reply by RED-CAD-Solutions-AG-Development** [reply]
Thank you
